server/getStores.ts

```diff
@@ -0,0 +1,9 @@
+"use server";
+
+import prisma from "@/lib/prisma";
+
+export async function getStores() {
+  const result = await prisma.store.findMany();
+  console.log(result);
```

Comment:
log not required

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/16/commits/74eef783e39164bdd19057fbd73f7d322ef53cbe